packages/nuxt/src/app/nuxt.ts

```diff
@@ -194,6 +194,10 @@ export function createNuxtApp (options: CreateOptions) {
     })
   }
 
+  // Log errors captured when running plugins, in the `app:created` and `app:beforeMount` hooks
+  // as well as when mounting the app and in the `app:mounted` hook
+  nuxtApp.hook('app:error', console.error)
```

Comment:
Tag was also nice idea btw before. Not sure if also makes negative DX, in this case feel free to ignore this 

```suggestion
  nuxtApp.hook('app:error', error => console.error('[nuxt] [app]', error))
```

Reply:
The reason I'm not including it is (1) I don't think we should persist `[nuxt]` logging tags into production, and (2) this isn't really an error that Nuxt is throwing unlike other errors tagged `[nuxt]`.

I did initially think it might improve the stacktrace but hookable gets in the way, so it doesn't make much difference for DX - we either get a breakpoint in hookable's `index.mjs` or here in `nuxt.mjs`.

Reply:
Mainly, this is to find the source of console errors (otherwise we are not sure if `console.error` is being called by user, nuxt or a library). And since this is a generic handler, i was thinking makes sense. I use this tags as standard method in unjs libraries too.

But also see that it might make another confusion that _nuxt_ is the source of the bug. What if make it more explicit: `Nuxt app caught error:`

Reply:
good idea!